src/open_inwoner/ssd/templates/jaaropgave.html

```diff
@@ -2,136 +2,142 @@
+			<title>Jaaropgave {{ spec.dienstjaar|slugify }}</title>
+	</head>
+	<body>
+	<div class="yearly-report">
+			<!-- HEADER -->
+			<!-- Title -->
+			<header class="header flex-container extra-margin-top">
+					<div class="flex-box heading">
+							<h1 class="h1">Jaaropgave {{ spec.dienstjaar|slugify }}</h1>
+					</div>
+					<div class="flex-box heading__logo right">
+							<img class="yearly-report__logo" src="{{ report.logo.url }}" alt="">
+					</div>
+			</header>
+
+			<!-- Addresses -->
+			<div class="flex-container__address address">
+					<div class="flex-box heading personalia">
+							<p class="yearly-report__p">{% format_string client.voorletters client.voorvoegsel client.achternaam %}</p>
+							<p class="yearly-report__p">{% format_string client.adres.straatnaam client.adres.huisnummer client.adress.huisletter %}</p>
+							<p class="yearly-report__p">{{ client.adres.postcode }} {{ client.adres.woonplaatsnaam }}</p>
+					</div>
+					<div class="flex-box heading address">
+							<div>
+									<p class="yearly-report__p inhoudingsplichtige">Inhoudingsplichtige</p>
+									<p class="yearly-report__p">Gemeente {{ inhoudingsplichtige.gemeentenaam }}</p>
+									<p class="yearly-report__p">{{ inhoudingsplichtige.bezoekadres }}</p>
+									<p class="yearly-report__p">{{ inhoudingsplichtige.postcode }} {{ inhoudingsplichtige.woonplaatsnaam }}</p>
+							</div>
+					</div>
+			</div>
+			<!-- PARTICIPATIEWET -->
+			<section class="section section__participatiewet">
+					<div class="heading">
+							<h2 class="h2">Participatiewet</h2>
+					</div>
+					<!-- row 1 -->
+					<div class="subsection">
+							<table class="yearly-report__table table-extra-col">
+									<colgroup>
+											<col style="width: 23%;"/>
+											<col style="width: 23%;"/>
+											<col style="width: 23%;"/>
+											<col style="width: 23%;"/>
+											<col style="width: 8%;"/>
+									</colgroup>
+									<tr class="yearly-report__tr">
+											<th class="yearly-report__th">Loon loonbelasting / volksverzekeringen</th>
+											<th class="yearly-report__th">Ingehouden Loonbelasting / Premie volksverzekeringen (loonheffing)</th>
+											<th class="yearly-report__th">Verrekende arbeidskorting</th>
+											<th class="yearly-report__th">Loonheffingskorting Met ingang van</th>
+											<th class="yearly-report__th">Code</th>
+									</tr>
+									<tr class="yearly-report__tr">
+											<td class="yearly-report__td">{% format_sign_value spec.fiscaalloon %}</td>
+											<td class="yearly-report__td">{% format_sign_value spec.loonheffing %}</td>
+											<td class="yearly-report__td">0</td>
+											<!-- Display first date -->
+											<td class="yearly-report__td">{% format_period spec.loonheffingskorting.0.ingangsdatum %}</td>
+											<td class="yearly-report__td">{{ spec.loonheffingskorting.0.cd_loonheffingskorting }}</td>
```

Comment:
The case of multiple dates is taken care of. I'm taking the first date first, then iterate over the rest (if there are any). I'm indeed assuming that there can't be zero dates for the report, but it will be better to do this in the template tag with basic error handling.

Reply:
I ended up rewriting the loop in the template.